lib/SILOptimizer/Utils/Local.cpp

```diff
@@ -2831,6 +2834,21 @@ bool swift::simplifyUsers(SILInstruction *I) {
   return Changed;
 }
 
+/// True if a type can be expanded
```

Comment:
git-clang-format? = ).

Reply:
did git-clang-format again - didn't complain about that one :)

Reply:
Ok. My bad. ; ).